src/azure-cli/azure/cli/command_modules/acr/_format.py

```diff
@@ -183,6 +185,12 @@ def _task_format_group(item):
         ('TRIGGERS', _get_triggers(item))
     ])
 
+def _taskrun_format_group(item):
```

Comment:
@huanwu some other properties to add similar to the standard run output would be 'RUN ID', 'STARTED', AND 'DURATION'

Reply:
I will add RUN ID but there're No Duration and Started properties in taskrun

Reply:
how do we know if run has started?

Reply:
The taskrun deployment won't return if run not finish. But for show/list/delete method, why do we care if run started or not?

Reply:
Duration might be a good thing to support in the future

Reply:
@shahzzzam @huanwu duration is caluculated from the `startTime` and `finishTime` of the completed task run, see how we calculate this for our current run output [run](https://github.com/Azure/azure-cli/blob/26c8ead7536cb3fe56490fd86db6b0a83b3c7066/src/azure-cli/azure/cli/command_modules/acr/_format.py#L210). These sort of metrics may be useful to customers so that they can track when a task was run and how long that run took to finish. Also has continuity with our current output format for runs.

Reply:
@jaysterp @shahzzzam Yes, the duration was added in the format calculated from startTime and finishTime.